packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -495,11 +500,51 @@ export default function PostOptionsMenu({
   postOptions.push({
     icon: <MenuIcon Icon={BlockIcon} />,
     label: isSourceBlocked
-      ? `Show posts from ${post?.source?.name}`
-      : `Don't show posts from ${post?.source?.name}`,
+      ? `Unblock ${post?.source?.name}`
+      : `Block ${post?.source?.name}`,
     action: isSourceBlocked ? onUnblockSourceClick : onBlockSourceClick,
   });
 
+  if (post?.author && post?.author?.id !== user?.id) {
+    postOptions.push({
+      icon: <MenuIcon Icon={BlockIcon} />,
+      label: isBlockedAuthor
+        ? `Unblock ${post.author.name}`
+        : `Block ${post.author.name}`,
```

Comment:
I think it should be different my feed vs custom feed but raised it here.
https://dailydotdev.slack.com/archives/C07GVUM114N/p1737018718495219?thread_ts=1736422027.072149&cid=C07GVUM114N